firmware/config/boards/kinetis/board.mk

```diff
@@ -3,15 +3,20 @@
 BOARD_DIR = $(PROJECT_DIR)/config/boards/$(PROJECT_BOARD)
 
 BOARDSRC = $(BOARD_DIR)/board.c
-BOARDSRC_CPP = $(BOARD_DIR)/board_configuration.cpp
+BOARDCPPSRC = $(BOARD_DIR)/board_configuration.cpp
 
 # Required include directories
-BOARDINC = $(BOARD_DIR)
+BOARDINC  = $(BOARD_DIR)
 BOARDINC += $(BOARD_DIR)/config/controllers/algo
 
-BOARDSRC_CPP += $(CONTROLLERS_ALGO_AUTO_GENERATED_ENUMS)
+BOARDCPPSRC += $(CONTROLLERS_ALGO_AUTO_GENERATED_ENUMS)
 
 # Define linker script file here
-LDSCRIPT= $(STARTUPLD)/MKE1xF512.ld
+LDSCRIPT = $(STARTUPLD)/MKE1xF512.ld
 
 CONFDIR = $(BOARD_DIR)
+
+# Shared variables
+ALLCSRC   += $(BOARDSRC)
```

Comment:
I see these lines repeated every time, why not either
- put them in `Makefile` or
- just `ALLSRC += $(BOARD_DIR)/board.c` etc

Reply:
I tried to keep unify with board.mk from ChibiOS.
When all .mk files will be modified this way Makefile can be simplified to something like:
`CSRC =  $(ALLCSRC)
CPPSRC = $(ALLCPPSRC)
INCDIR = $(ALLINC)`